### PR TITLE
Content edits for 22nd March 2024 workshop

### PR DESCRIPTION
**Pull request by alishamohanty**
Content edits for 22nd March 2024 workshop.
Suggested changes are elaborated in this slack [message](https://inclojureorg.slack.com/archives/CCDQALNDB/p1706184482937019).

### CODE REVIEW

File: content.org [hunk 1]
```diff
@@ -31,8 +24,21 @@
...
+
+*** Soumya Maithani
+Currently working as a senior platform developer at Helpshift, developing
+in Clojure for the past 9 years and hoping to continue for more.
+I love distributed systems, event streaming and exploring new databases.
+In my spare time, I love reading fiction, cooking and am learning football.
+
+*** Sezal Jain
+
+# TODO
```
Comment (PriyangaPKini) [inline content.org]: Intro for Sezal is pending

**Comment by PriyangaPKini** [comment]
hey @alishamohanty, Please provide a description for this PR.

**Comment by PriyangaPKini** [comment]
Also, are we going to share this repo with the workshop attendees? Or is this an internal reference of what we will be doing in the workshop?

File: content.org (hunk 1 above)
Comment (alishamohanty) [reply content.org]: Yeah, I haven't received a bio from her yet.

**Comment by alishamohanty** [comment]
> Also, are we going to share this repo with the workshop attendees? Or is this an internal reference of what we will be doing in the workshop?

 Yes, we will share this repo with the attendees.

File: content.org [hunk 2]
```diff
@@ -1,24 +1,17 @@
...
-  years of my engineering career I have seen the disparity in the
-  gender ratio at all the companies I have worked with. In some teams,
-  I was the only female developer. In some teams, I was the only
-  female. I have never had a female mentor before. But, I have also
-  seen companies change, grow, become more aware of this issue, and
-  find ways to fix it. This is a small attempt towards that. Getting
-  more females to learn programming, learn Clojure and do awesome
-  things that come with ability to command a computer.
+- This workshop is aimed specifically at women and non-binary people. 
+  This is a small attempt towards that. Getting more females to learn
```
Comment (qptr) [inline content.org]: The first two sentences are disjointed.

Comment (qptr) [reply content.org]: Can we reword this paragraph? I don't follow what it is trying to say

File: content.org [hunk 3]
```diff
@@ -31,8 +24,21 @@
 
 - Also, thanks to nilenso, IN/Clojure and ClojureBridge core team.
 
-- https://tinyurl.com/clj-bridge-blr
--> outline.org
+- https://nilenso.com/clojurebridge
```
Comment (qptr) [inline content.org]: This link is dead. What is this supposed to point to?

File: content.org [hunk 4]
```diff
@@ -110,6 +116,11 @@
 - Let's start a REPL. and we can type and evaluate things as we go on
   learning them.
 
+- How to open a repl in VSCode 
+  - Open command using: `ctrl+shift+P` or `cmd+shift+P`
```
Comment (qptr) [inline content.org]:
```suggestion
  - Open command palette using: `ctrl+shift+P` or `cmd+shift+P`
```

File: content.org [hunk 5]
```diff
@@ -110,6 +116,11 @@
 - Let's start a REPL. and we can type and evaluate things as we go on
   learning them.
 
+- How to open a repl in VSCode
```
Comment (qptr) [inline content.org]:
```suggestion
- How to open a REPL in VSCode
```

File: content.org [hunk 6]
```diff
@@ -239,6 +250,25 @@ Evaluate some normal functions in your REPL
 (,+ 1 2)
 #+END_SRC
 
+**** Structural Editing
```
Comment (qptr) [inline content.org]: Solid section

File: content.org [hunk 7]
```diff
@@ -239,6 +250,25 @@ Evaluate some normal functions in your REPL
...
+
+Structural editing is a method of manipulating code based on its syntactic structure, 
+rather than its text representation.
+It allows for operations like moving, wrapping or editing code blocks as discrete units.
+
+Plugins that enable structural editing:
+
+Visual studio code: Calva (https://calva.io/)
+Intellij: Cursive (https://cursive-ide.com/)
```
Comment (qptr) [inline content.org]: Bullets so its clearer to see
```suggestion
Plugins that enable structural editing:

- Visual Studio Code: Calva (https://calva.io/)
- Intellij: Cursive (https://cursive-ide.com/)
```

File: content.org (hunk 6 above)
Comment (qptr) [reply content.org]: Can we move this part to below the whole Expressions section? It breaks it up while reading so the notation part of Expressions is below Strutural Editing. I know this is where structural editing becomes relavant. Maybe we can add a note saying `See below: Structural Editing` so it reads smoother.

Right now, it looks like Structural editing, Infix notation, Prefix notation are all subheadings under Expressions.

Some kind of formatting to set it apart form the learning material would help with this too, something like [this](https://github.blog/changelog/2023-12-14-new-markdown-extension-alerts-provide-distinctive-styling-for-significant-content/)? Leaving it upto you

File: content.org [hunk 8]
```diff
@@ -611,7 +649,7 @@ List of basic types:
 - first returns the first item in the collection.
 #+BEGIN_SRC clojure
 (first [5 10 15])
-(nth [5 10 15] 0)
+;; TODO write using nth function?
```
Comment (qptr) [inline content.org:663]: I think we can keep this around. Seems like it's just to reinforce how nth works. Applying something we learned right away in a different context.

File: resources.org [hunk 9]
```diff
@@ -0,0 +1 @@
+Clojure learning material;s curated list by Srihari: https://gist.github.com/ssrihari/0bf159afb781eef7cc552a1a0b17786f
```
Comment (qptr) [inline resources.org]: Stray semicolon

Comment (qptr) [reply resources.org]: Think this page can use formatting
```suggestion
# More Clojure learning resources

[Curated list by Srihari](https://gist.github.com/ssrihari/0bf159afb781eef7cc552a1a0b17786f)
```

**Review by qptr — CHANGES_REQUESTED** [review]
Mostly looks solid.

Why have we removed all the exercises from this repo? What's the plan?

File: content.org [hunk 10]
```diff
@@ -31,8 +24,21 @@
 
 - Also, thanks to nilenso, IN/Clojure and ClojureBridge core team.
 
-- https://tinyurl.com/clj-bridge-blr
--> outline.org
+- https://nilenso.com/clojurebridge
+
+** Speakers
+
+*** Soumya Maithani
```
Comment (qptr) [inline content.org]: Typo: Somya

File: content.org (hunk 6 above)
Comment (alishamohanty) [reply content.org]: I have moved "Structural Editing" section post "Expression" section.

File: content.org (hunk 8 above)
Comment (alishamohanty) [reply content.org:663]: Yes, but we will ask this to the audience. Thats why I have removed the solution.

Comment (alishamohanty) [reply content.org:652]: Yes, but we will ask this to the audience. Thats why I have removed the solution.

File: content.org (hunk 3 above)
Comment (alishamohanty) [reply content.org]: I will redirect to the github repo instead.

File: content.org (hunk 1 above)
Comment (alishamohanty) [reply content.org]: Have added a bio.

**Comment by alishamohanty** [comment]
> Mostly looks solid.
> 
> Why have we removed all the exercises from this repo? What's the plan?

We haven't removed any exercises. They still exist in the exercises folder. We have only removed the mars rover problem which is not a part of the curriculum due to limited time constraint. We will only solve Land of Wizards problem during the workshop.

File: content.org [hunk 11]
```diff
@@ -1,24 +1,16 @@
...
-  years of my engineering career I have seen the disparity in the
-  gender ratio at all the companies I have worked with. In some teams,
-  I was the only female developer. In some teams, I was the only
-  female. I have never had a female mentor before. But, I have also
-  seen companies change, grow, become more aware of this issue, and
-  find ways to fix it. This is a small attempt towards that. Getting
-  more females to learn programming, learn Clojure and do awesome
-  things that come with ability to command a computer.
+- This workshop is specifically designed for women and non-binary individuals. 
+It's a small effort to encourage more females to learn programming, particularly
```
Comment (sezaljain) [inline content.org:5]: 'more females' -> them

Comment (sezaljain) [reply content.org:5]: acutally, I would suggest to keep the original content

File: content.org [hunk 12]
```diff
@@ -284,6 +297,26 @@ Evaluate some normal functions in your REPL
...
+ - Intellij: Cursive (https://cursive-ide.com/)
+
+Common Paredits for Visual Studio: 
+
+| Action                     | Windows                | Linux      | Mac                     |
+| -------------------------- | ---------------------- | ---------- | ------------------------|
+| Slurp Forward              | ctrl+alt+right         | ctrl+alt+. | ctrl+opt+right          |
+| Barf Forward               | ctrl+alt+left          | ctrl+alt+, | ctrl+opt+left           |
+| Raise Sexp                 |       ctrl+alt+p  ctrl+alt+r        | ctrl+opt+p  ctrl+opt+r  |
+| Drag sexp forward/backward |           alt+up/down arrows        | opt+up/down arrows      |
```
Comment (sezaljain) [inline content.org]: try fixing the table here by writing something like
| Raise Sexp                 |       ctrl+alt+p  ctrl+alt+r       || ctrl+opt+p  ctrl+opt+r  |

**Review by sezaljain — APPROVED** [review]

**Pull request merged**